orchestration/orchestration.go

```diff
@@ -180,15 +180,41 @@ func (o *Orchestrator) DeleteService(ctx context.Context, input *ServiceDeleteIn
 func (o *Orchestrator) processCluster(ctx context.Context, input *ServiceOrchestrationInput) (*ecs.Cluster, error) {
 	client := o.ECS
 	if input.Service.Cluster != nil {
-		log.Infof("Using provided cluster name %s", aws.StringValue(input.Service.Cluster))
+		log.Infof("Using provided cluster name %s", aws.StringValue(input.Cluster.ClusterName))
+
 		cluster, err := getCluster(ctx, client, input.Service.Cluster)
 		if err != nil {
 			return nil, err
 		}
+
 		log.Debugf("Got cluster %+v", cluster)
 		return cluster, nil
 	} else if input.Cluster != nil {
 		log.Infof("Creating cluster %s", aws.StringValue(input.Cluster.ClusterName))
+
+		tagCount := 0
+		for _, tag := range input.Cluster.Tags {
+			// the tag is set and not set to our Org, rewrite it
+			if aws.StringValue(tag.Key) == "spinup:tag" {
+				if aws.StringValue(tag.Value) != *aws.String(Org) {
```

Comment:
`*aws.String(Org)`

this gets the creates a pointer from Org and then gets the value at the pointer